Ada Examples/Examples Based On OpenGL Superbible/Simplest/src/a_dot.adb

```diff
@@ -0,0 +1,48 @@
+-- Program A_Dot
+-- Author Roger Mc Murtrie
+-- Created 16 December 2016
+
+with Ada.Directories; use Ada.Directories;
+with Ada.Exceptions; use Ada.Exceptions;
+with Ada.Text_IO; use Ada.Text_IO;
+
+with GL.Objects.Programs;
+with GL.Objects.Vertex_Arrays;
+
+with glfw;
+
+with Initialization;
+with Run;
+with Utilities;
+with Window_Types;
+
+procedure A_Dot is
+
+    Default_Directory           : String := ".";
+
+    Main_Window             : Window_Types.tWindow;
+    Rendering_Program   : GL.Objects.Programs.Program;
+    Vertex_Array              : GL.Objects.Vertex_Arrays.Vertex_Array_Object;
+begin
+    -- Set the base directory for the executing program
+    Set_Directory (Default_Directory);
```

Comment:
This essentially does nothing since `"."` denotes the current directory. So you are changing the current directory to itself. You can just remove that line.